### PR TITLE
update the pnpmfile.cjs

### PR DESCRIPTION
**Pull request by aruniverse**
update the pnpmfile.cjs, so all external pkgs to the repo which have dependencies found in this one all resolve to the same pkg. 

previously youd have another instance of core-{bentley, geometry, frontend} in your tree

### CODE REVIEW

File: common/config/rush/.pnpmfile.cjs [hunk 1]
```diff
@@ -1,29 +1,54 @@
...
     pkg.dependencies["@itwin/core-bentley"] = "workspace:*";
   } else if (pkg.name == "@itwin/electron-authorization") {
     pkg.dependencies["@itwin/core-bentley"] = "workspace:*";
+  } else if (pkg.name == "@itwin/oidc-signin-tool") {
+    pkg.dependencies["@itwin/core-bentley"] = "workspace:*";
   }
+
+  // https://github.com/iTwin/projects-client
+  if (pkg.name == "@itwin/projects-client") {
+    pkg.dependencies["@itwin/core-bentley"] = "workspace:*";
```
Comment (calebmshafer) [inline common/config/rush/.pnpmfile.cjs:38]: There shouldn't be a real dependency in the projects-client. Are you seeing issues?

Comment (aruniverse) [reply common/config/rush/.pnpmfile.cjs]: Ah good catch, its just a dev dep in there. Will remove.

File: common/config/rush/.pnpmfile.cjs [hunk 2]
```diff
@@ -1,29 +1,49 @@
...
+
   // Hacky mess: For external packages to this monorepo that have peer dependencies on packages
   // in this repo, we need to do some magic in order to get the peerDeps to point to a correct
   // version of the packages. Update the pkg.json real dependency list to
   // Note that these dependencies are only ever allowed for testing purposes and should not be the
   // dependency of any published packages.
-  else if (pkg.name == "@itwin/imodels-access-backend") {
+
+  // https://github.com/iTwin/imodels-clients
+  if (pkg.name == "@itwin/imodels-access-backend") {
```
Comment (calebmshafer) [inline common/config/rush/.pnpmfile.cjs:17]: one last nitpick: why did you change these to different if/else blocks instead of keeping the ifelse syntax? The `pkg.name` is only going to ever be valid for one of these names so if it's already matched one no need to continue checking.

Comment (aruniverse) [reply common/config/rush/.pnpmfile.cjs:17]: you're right, just a minor personal stylistic preference, thought it was better to create a new if/else block for new repos, but will revert.

Comment (calebmshafer) [reply common/config/rush/.pnpmfile.cjs]: thanks :)

**Review by calebmshafer — APPROVED** [review]
